app/src/angular/services/common.service.js

```diff
@@ -63,16 +63,21 @@ function CommonService($rootScope, $log, $q, $mdDialog, $compile, $mdToast) {
             return result;
         }
 
-        commasAfterNumber (amount, num) {
+        commasAfterNumber(amount, num) {
```

Comment:
Why are there two declarations of the same function name in one file? See line 48.

Reply:
don't know who worked(used) on that(first one),maybe some merge problem.I commented first one in case someone still needs that.